index.js

```diff
@@ -16,41 +13,44 @@ var typedArrays = {
 	Uint16Array: true,
 	Uint32Array: true
 };
-
 var slice = String.prototype.slice;
-var toStrTags = {};
+var getpr = Object.getPrototypeOf;
 var gOPD = Object.getOwnPropertyDescriptor;
-if (hasToStringTag && gOPD && Object.getPrototypeOf) {
-	forEach(typedArrays, function (_, typedArray) {
-		var arr = new global[typedArray]();
-		if (!(Symbol.toStringTag in arr)) {
-			throw new EvalError('this engine has support for Symbol.toStringTag, but ' + typedArray + ' does not have the property! Please report this.');
+var isTypedArray;
+if (getpr && gOPD) {
+	if (typeof Symbol === 'function') {
+		if (typeof Symbol.toStringTag === 'symbol') {
```

Comment:
let's de-nest this by restoring the `var hasToStringTag = typeof Symbol === 'function' && typeof Symbol.toStringTag === 'symbol';` line